libecl/include/ert/ecl/ecl_nnc_export.h

```diff
@@ -38,6 +38,8 @@ typedef struct {
   int grid_nr2;
   int global_index2;
 
+  int nnc_index;
```

Comment:
Yes - I know that name was already used, so in that sense your choice was the natural one, I would still prefer the change though.